modules/EasyAbp.EShop.Orders/src/EasyAbp.EShop.Orders.Application/EasyAbp/EShop/Orders/Orders/CurrencyIsLimitException.cs

```diff
@@ -5,10 +5,9 @@ namespace EasyAbp.EShop.Orders.Orders
 {
     public class CurrencyIsLimitException : BusinessException
     {
-        public CurrencyIsLimitException(string expectedCurrency) : base(
-            "CurrencyIsLimit",
-            $"Only the specified currency {expectedCurrency} is allowed.")
+        public CurrencyIsLimitException(string expectedCurrency) : base(OrdersErrorCodes.CurrencyIsLimit)
```

Comment:
Please change this error name to "UnexpectedCurrency" for me. Thanks.

Reply:
I think we can keep it for now.

Reply:
done